apps/console/src/features/connections/components/edit/forms/authenticators/email-otp-authenticator-form.tsx

```diff
@@ -99,7 +99,7 @@ interface EmailOTPAuthenticatorFormInitialValuesInterface {
     /**
      * Allow OTP token to have 0-9 characters only.
```

Comment:
Let's update the TSDoc comments too. Please check the other places as well.

Reply:
Resolve with 41555a63bde603bb36b8c7b0c1acf48eea654946